fuel-vm/src/interpreter/alu/muldiv.rs

```diff
@@ -16,29 +16,17 @@ where
         rhs: Word,
         divider: Word,
     ) -> Result<(), RuntimeError> {
-        let (
-            SystemRegisters {
-                flag,
-                mut of,
-                mut err,
-                pc,
-                ..
-            },
-            mut w,
-        ) = split_registers(&mut self.registers);
-        let dest = &mut w[ra.try_into()?];
```

Comment:
Hmm, I don't know how it worked before, but for me `w[ra]`(where `w` is a writable registers without system registers) and `self.registers[ra]`(it is system registers and writable together) are different things. Because the lengths of arrays are different and with the same `ra`, they will point o different registers.

Is that a bug? If yes, what is the right version?